utils/releaser/src/ReleaseWorker/Release/CreateAndCommitComposerLockAndPackageLockReleaseWorker.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\Releaser\ReleaseWorker\Release;
+
+use PharIo\Version\Version;
+use Shopsys\Releaser\ReleaseWorker\AbstractShopsysReleaseWorker;
+use Shopsys\Releaser\Stage;
+
+final class CreateAndCommitComposerLockAndPackageLockReleaseWorker extends AbstractShopsysReleaseWorker
+{
+    /**
+     * @param \PharIo\Version\Version $version
+     * @return string
+     */
+    public function getDescription(Version $version): string
+    {
+        return 'Create and commit composer.lock and package-lock.json and [Manually] push it';
+    }
+
+    /**
+     * Higher first
+     * @return int
+     */
+    public function getPriority(): int
+    {
+        return 630;
+    }
+
+    /**
+     * @param \PharIo\Version\Version $version
+     */
+    public function work(Version $version): void
+    {
+        $this->processRunner->run('rm -rf project-base/vendor');
```

Comment:
When it started running, I thought it was an error and I have to do something manually. After 10 seconds, the PC fan started to run and I realized it probably does something.

Please add here an information that it is running, optionally animation so I won't be tempted to do something

Reply:
I added a note as well

Reply:
The note is a bit ill-structured, but fine.

I'd use a list: *Removing vendor/, node_modules/, composer.lock, and package-lock.json* instead of *Removing vendor and node_modules folders and composer.lock and package-lock.json*...

Reply:
I changed it to what you wrote.

Reply:
You forgot the [oxford comma](https://en.wikipedia.org/w/index.php?title=Oxford_comma).